src/apply_rules.rs

```diff
@@ -110,18 +209,137 @@ fn insert_code<T>(
         .insert((code.object_system, code.object_code));
 }
 
+fn update_prop<T: Clone + From<String> + Into<Option<String>>>(
+    p: &PropertyRule,
+    field: &mut T,
+    report: &mut Report,
+) {
+    let any_prop = Some("*".to_string());
+    if p.property_old_value == any_prop || p.property_old_value == field.clone().into() {
+        *field = T::from(p.property_value.clone());
+    } else {
+        report.add_warning(
+            format!(
+                "object_type={}, object_id={}, property_name={}: property_old_value does not match the value found in the data",
+                p.object_type.as_str(),
+                p.object_id,
+                p.property_name
+            ),
+            ReportType::OldPropertyValueDoesNotMatch,
+        );
+    }
+}
+
+fn wkt_to_geo(wkt: &str, report: &mut Report, p: &PropertyRule) -> Option<GeoGeometry<f64>> {
+    if let Ok(wkt) = wkt::Wkt::from_str(wkt) {
+        if let Ok(geo) = try_into_geometry(&wkt.items[0]) {
+            Some(geo)
+        } else {
+            warn!("impossible to convert empty point");
```

Comment:
why would this be a point ?

Reply:
the only time the conversion to wkt -> geo fails is when wkt = "EMPTY POINT" but the library doesn't know how to handle this case because it uses rust-geo which doesn't know how to handle empty points either